src/item.h

```diff
@@ -2058,9 +2059,12 @@ class item : public visitable<item>
         char invlet = 0;      // Inventory letter
         bool active = false; // If true, it has active effects to be processed
         bool is_favorite = false;
+        mutable std::map<clothing_mod_type, float> clothing_mod_val_cache;
```

Comment:
And instead of adding yet another member to `item` (which has to be copied, understand, manged, ...), you could store your values in `item::item_vars` (accessible via `item::has_var`, `item::set_var` and `item::get_var`).

Reply:
`item::set_var` is non-const function, so it is needed that making many const functions non-const function. (e.g. `item::bash_resist`) I think that brings very large impact.